apps/kampus/app/pano/@modal/(.)post/create/page.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
```

Comment:
first time seeing @page and "(.)page" is it just naming or does it do something on next router ?